cli/util/upload_bin.go

```diff
@@ -285,11 +285,7 @@ func searchIndexFile(ctx *cli.Context, p poolWrapper, containerID cid.ID, privKe
 			for range indexIDs {
 				count++
 				if count > 1 {
-					select {
-					case errCh <- fmt.Errorf("duplicated index file %d found", i):
-					default:
-					}
-					return
+					fmt.Fprintf(ctx.App.Writer, "WARN: For index %d found %d index files\n", i, count)
```

Comment:
It will print the log as many times as the number of duplicates. Move this out of cycle, and attach OIDs to the error message.